frontend/src/components/common/modal/Modal.tsx

```diff
@@ -18,6 +19,7 @@ import ModalText from "./ModalText";
 interface ModalProps {
   isOpen: boolean;
   onClose: () => void;
+  variant?: "default" | "large";
```

Comment:
`size` might be more fitting? but not sure, cause we use this value in other contexts like the divider thing

 and with that, does Chakra modal size help with setting the width?